src/nvim/buffer_defs.h

```diff
@@ -649,7 +649,7 @@ struct file_buffer {
   bool b_p_swf;                 /* 'swapfile' */
   long b_p_smc;                 /* 'synmaxcol' */
   char_u      *b_p_syn;         /* 'syntax' */
```

Comment:
You will clearly have to convert *all* comments here from `/*` to `//`: adjacent lines, linter complains. Better in a separate commit. Also making them look like `///< 'syntax' option value` is probably more appropriate.